common/sgx/tlsverifier.c

```diff
@@ -172,3 +197,121 @@ oe_result_t oe_verify_attestation_certificate(
     oe_free(report);
     return result;
 }
+
+/**
+ * oe_verify_plugin_attestation_certificate
+ *
+ * This function perform a custom validation on the input certificate. This
+ * validation includes extracting an attestation evidence extension from the
+ * certificate before validating this evidence. An optional
+ * claim_verify_callback could be passed in for a calling client to further
+ * validate the claims of the enclave creating the certificate.
+ * @param[in] cert_in_der a pointer to buffer holding certificate contents
+ *  in DER format
+ * @param[in] cert_in_der_len size of certificate buffer above
+ * @param[in] claim_verify_callback callback routine for custom claim checking
+ * @param[in] arg an optional context pointer argument specified by the caller
+ * when setting callback
+ * @retval OE_OK on a successful validation
+ * @retval OE_VERIFY_FAILED on quote failure
+ * @retval OE_INVALID_PARAMETER At least one parameter is invalid
+ * @retval OE_FAILURE general failure
+ * @retval other appropriate error code
+ */
+oe_result_t oe_verify_plugin_attestation_certificate(
+    uint8_t* cert_in_der,
```

Comment:
Coding style nit: expand "cert" -> "certificate" throughout.  Similarly "buf" -> "buffer"

Reply:
The sister functions such as `oe_verify_attestation_certificate` use `cert` and `buf` throughout. I prefer to keep the terms consistent.